File: composeApp/src/androidMain/kotlin/org/hossian/cryptoappkmp/coin_list/CoinListState.kt
package org.hossian.cryptoappkmp.coin_list

import org.hossian.cryptoappkmp.domain.model.Coin

data class CoinListState(
    val isLoading: Boolean = false,
    val coins: List<Coin> = emptyList(),
    val error: String = ""
)

File: iosApp/iosApp/coin_list/CoinListState.swift
//
//  CoinListState.swift
//  CryptoApp.ios
//
//  Created by Hossain Muktar on 7/8/24.
//

import Foundation
import Shared

struct CoinListState {
    var isLoading: Bool = true
    var coins: [Coin]? = nil
    var error: String? = nil
}
